scss/custom/_card.scss

```diff
@@ -49,6 +49,11 @@
   }
 }
 
+.card-body-slim {
+  flex: 1 1 auto;
```

Comment:
@I2olanD Why do we need the flex here? And what do you think about moving the card-body-slim into the card-body class since it is part of it, a modifier?

Reply:
@aholzner this is just a modification of the original bootstrap card. it is no modifier anyways. it is more described as a "slim" version of the card-body

Reply:
@I2olanD If I understand correctly, a describe is a modification. So, we can either use the card, with it is default out-of-the-box behavior/shape, or use the slim version of it. What do you think?

Reply:
@aholzner i don't really understand what you mean. In my opinion this class describes a completely different thing than a card-body.